docs/walrus-sites/portal.md

```diff
@@ -11,6 +11,10 @@ portals:
 Currently, the server-side and service-workers portals are available at <https://blob.store> and
 <https://walrus.site>, respectively.
 
+```admonish danger title="Walrus Sites stable branch"
+The stable branch of Walrus Sites is `testnet`.
+```
+
 ## Running the portal locally
```

Comment:
Suggestion: this is fine as it is, but in a follow up could you document the two different portals and the use of the new flags @Tzal3x ?

Reply:
Tracked in SEINT-231. ✅